src/Generating/StructGen.cpp

```diff
@@ -456,38 +456,38 @@ void cStructGenDirectOverhangs::GenFinish(cChunkDesc & a_ChunkDesc)
 	int BaseY = 63;
 
 	// Interpolate the lowest floor:
-	for (int z = 0; z <= 16 / INTERPOL_Z; z++) for (int x = 0; x <= 16 / INTERPOL_X; x++)
+	for (int z = 0; z <= cChunkDef::Width / INTERPOL_Z; z++) for (int x = 0; x <= cChunkDef::Width / INTERPOL_X; x++)
 	{
 		FloorLo[INTERPOL_X * x + 17 * INTERPOL_Z * z] =
 			m_Noise1.IntNoise3DInt(BaseX + INTERPOL_X * x, BaseY, BaseZ + INTERPOL_Z * z) *
 			m_Noise2.IntNoise3DInt(BaseX + INTERPOL_X * x, BaseY, BaseZ + INTERPOL_Z * z) /
-			256;
+			cChunkDef::VerticalBlockCount;
```

Comment:
```suggestion
			256;
```
This is noise-scaling, not coords.